database/scripts/20220718_210029_fecha_estimada_ofertas_down.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE plan_adquisiciones."Registro_plan_adquisiciones"
+DROP COLUMN IF EXISTS fecha_estimada_ofertas_inicio;
+
+ALTER TABLE plan_adquisiciones."Registro_plan_adquisiciones"
+DROP COLUMN IF EXISTS fecha_estimada_ofertas_fin;
```

Comment:
Dejar vacía la última línea...
```suggestion
DROP COLUMN IF EXISTS fecha_estimada_ofertas_fin;

```